Sources/Orbit/Components/ListChoice.swift

```diff
@@ -0,0 +1,252 @@
+import SwiftUI
+
+/// Shows one of a selectable list of items with similar structures.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/listchoice/)
+/// - Important: Component expands horizontally to infinity up to a ``Layout/readableMaxWidth``.
+public struct ListChoice: View {
+
+    let title: String
+    let description: String
+    let icon: Icon.Symbol
+    let trailingElement: ListChoice.TrailingElement
+    let backgroundColor: BackgroundColor?
+    let action: () -> Void
+
+    public var body: some View {
+        SwiftUI.Button(
+            action: {
+                HapticsProvider.sendHapticFeedback(.light(0.5))
+                action()
+            },
+            label: {
+                buttonContent
+            }
+        )
+        .buttonStyle(ListChoiceButtonStyle(backgroundColor: backgroundColor))
+        .accessibility(label: SwiftUI.Text(title))
+        .accessibility(hint: SwiftUI.Text(description))
+    }
+
+    @ViewBuilder var buttonContent: some View {
+        HStack(spacing: 0) {
+            header
+                .frame(maxWidth: .infinity, alignment: .leading)
+
+            // Provide minimal height (frame(minHeight:) collapses multiline text in snapshots)
+            Color.clear
+                .frame(width: 0, height: 48)
+
+            trailingElementView
+                .padding(.trailing, .medium)
+        }
+        .frame(maxWidth: Layout.readableMaxWidth, alignment: .leading)
+        .overlay(separator, alignment: .bottom)
+    }
+    
+    @ViewBuilder var header: some View {
+        if isHeaderEmpty == false {
+            HStack(alignment: .firstTextBaseline, spacing: .small) {
```

Comment:
It might be possible to use the `Header` component for this whole header. It has more correct and unified alignment of icon (plus auto-sizing of icon, based on title style).

The `Header` would need to support a non-Heading style, though. In this case, the large+medium text, rather than actual Heading style.

In case of not using the `Header` yet, the `Text(title)` can be simply updated to `Text(title, weight: .medium)`.

Reply:
Hmm I was sure that I checked the title should be `regular`, but you're right. I'll take the easier way out and just add the `weight` for now.